mappings/markdown/Zeek/conn_log/README.md

```diff
@@ -29,6 +29,9 @@
 | OCSF                          | Raw           |
 | ----------------------------- | --------------|
 |`time`                         |`ts`           |
+|`start_time`                   |`ts`           |
+|`metadata.loggers[].logged_time`|`_write_ts`   |
```

Comment:
I would suggest `metadata.logged_time` for `_write_ts`.

In general, the `loggers` array is useful when there is a chain of tools that got the event up to this point. I'm skeptical that putting the tool that generated the log there, but I'm not seeing an alternative for `_system_name` either. Maybe ask in Slack if there's an attribute for the device where the log originated? Of course we have the host profile, but that's not so much about the log metadata but rather its contents.

Reply:
That seems exactly right.  I've updated each PR accordingly.